apps/console/src/app/components/prompts/views/PromptEditView.tsx

```diff
@@ -66,7 +67,7 @@ export const PromptEditView = () => {
       : currentPromptVersion.settings,
     content: isDraft
       ? getDraftPromptData(prompt.integrationId).content
-      : currentPromptVersion.content,
+      : currentPromptVersion.settings.modelSettings.messages?.[0].content,
   };
```

Comment:
Refer to what I wrote above about prompt testing. If we don't support it in this release, let's get rid of code references